services/tables/src/main/java/com/linkedin/openhouse/tables/api/validator/impl/OpenHouseTablesApiValidator.java

```diff
@@ -141,6 +143,11 @@ private void validatePolicies(CreateUpdateTableRequestBody createUpdateTableRequ
                   "%s : %s",
                   policiesSpecValidator.getField(), policiesSpecValidator.getMessage())));
     }
+    if (createUpdateTableRequestBody.getPolicies() != null
+        && createUpdateTableRequestBody.getPolicies().getReplication() != null) {
+      replicationConfigValidator.validate(
```

Comment:
Need to throw exception if validation Fails to ensure exception is propagated to controller layer. Looks like it got missed. We can have `replicationConfigValidator` follow similar pattern to `policiesSpecValidator` for validation logic and exception handling logic